.github/workflows/release-cli.yaml

```diff
@@ -53,23 +72,24 @@ jobs:
 
   sync-plugins:
     runs-on: ubuntu-20.04
-    needs: create-release
+    needs: [push-tag, create-release]
     steps:
       - name: Checkout buildbuddy-io/buildbuddy
         uses: actions/checkout@v3
         with:
           path: buildbuddy
-          ref: cli-v${{ needs.create-release.outputs.VERSION }}
+
       - name: Checkout buildbuddy-io/plugins
         uses: actions/checkout@v3
         with:
           repository: buildbuddy-io/plugins
           path: plugins
           token: ${{ secrets.BUILDBUDDY_GITHUB_USER_TOKEN }}
+
       - name: Sync plugins and push version tag
         env:
           GITHUB_TOKEN: ${{ secrets.BUILDBUDDY_GITHUB_USER_TOKEN }}
-          VERSION: ${{ needs.create-release.outputs.VERSION }}
+          VERSION: $${{ needs.push-tag.outputs.version }}
```

Comment:
is the `$$` intentional? (here and below)

Reply:
No thank you - typo